code-challenges.js

```diff
@@ -31,24 +56,62 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+//Function
+const boilingPoint = (temp1) => {
+    if(temp1 > 212){
+        return `${temp1} is above boiling point`
+    } else if(temp1 == 212){
+        return `${temp1} is at boiling point`
+    } else {
+        return `${temp1} is below boiling point`
+    }
+
+}
+//console.log(boilingPoint(temp1))
+//output: 42 is below boiling point
+
+//console.log(boilingPoint(temp2))
+//output: 350 is above boiling point
+
+//console.log(boilingPoint(temp3))
+//output: 212 is at boiling point
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+//create an array called padresScore that uses the built-in-method .concat to put different strings called padres1984WorldSeriesRuns and padres1998WorldSeriesRuns combines them. The output will be the number when combining the arrays.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+//Function
+let padresScore = (padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+
+//console.log(padresScore.length)
+//output: 9
```

Comment:
Well done!  You can even chain .length to the end of your concat statement, then just console.log(padresScore)!  You're doing great.